### DISCUSSION TITLE
Create a contact form with Nuxt and Strapi

### DISCUSSION THREAD

**Discussion by Jesus82** [strapi/strapi#7569] [Legacy Topics]
Hi all,

I'm trying to create a simple "Contact Us" form using strapi and nuxt in the frontend. So far I have installed "strapi-provider-email-nodemailer", and added this in `/config/plugins.js`
```

module.exports = ({ env }) => ({
  email: {
    provider: 'nodemailer',
    providerOptions: {
      host: env('SMTP_HOST', 'smtp.example.com'),
      port: env('SMTP_PORT', 587),
      auth: {
        user: env('SMTP_USERNAME'),
        pass: env('SMTP_PASSWORD'),
      }
      // ... any custom nodemailer options
    },
    settings: {
      defaultFrom: 'xxxxxx@ethereal.email',
      defaultReplyTo: 'xxxx@example.com',
    }
  }
});
```

I'm trying to test this setup with postman, but when a send a request like this

```
{   
    "subject":"prueba",
    "text":"Prueba"
}
```

It gives me `Error: Couldn't send email: No recipients defined.`

Any idea how to move forward or any other recommended settings for what I'm trying to achieve? Thanks in advance!

**Comment by derrickmehaffy** [comment]
Can you please clarify what endpoint you are hitting to test? Based on that error there is no "to" email defined hence the error of `No recipients defined` as you haven't set a recipient email.

**Reply by Jesus82** [reply]
endpoint: http://localhost:1337/email/

I also tried including a "to:xxxx.mail.com" but the error was the same
![captura](https://user-images.githubusercontent.com/8875832/91135851-703d8780-e6ae-11ea-85bb-0872cfe65761.png)

**Reply by derrickmehaffy** [reply]
Hi @Jesus82 I installed nodemailer (as this package wasn't created by nor maintained by the Strapi team, I can't provide direct support for it).

And just created a custom controller to handle sending mail:

![image](https://user-images.githubusercontent.com/8593673/91205240-1347ce80-e6ba-11ea-84d9-0029b4275ec0.png)

And the mail was sent fine (I received it in my inbox):
![image](https://user-images.githubusercontent.com/8593673/91205284-28246200-e6ba-11ea-88fe-1e1c5104cf2f.png)
![image](https://user-images.githubusercontent.com/8593673/91205544-936e3400-e6ba-11ea-929e-2a4c776c4d8d.png)

**Reply by derrickmehaffy** [reply]
You could auto-populate the `from` from the plugin settings but in this case I just sent it myself
